packages/server/src/api/bannerRouter.ts

```diff
@@ -117,8 +142,11 @@ export const buildBannerRouter = (
                 bannerChannel: test.bannerChannel,
                 isSupporter: !targeting.showSupportMessaging,
                 countryCode: targeting.countryCode,
-                content: variant.bannerContent,
-                mobileContent: variant.mobileBannerContent,
+                content: variant.bannerContent === null ? variant.bannerContent : bannerContent, // TODO: refactor this so that you don't have to pass through an explicit null
+                mobileContent:
+                    variant.mobileBannerContent === null
+                        ? variant.mobileBannerContent
+                        : bannerMobileContent, // TODO: refactor this so that you don't have to pass through an explicit null
```

Comment:
Thumbs up for the refactor note here. We shouldn't have to pass through null explicitly as a value here